client/src/pages/SignUp/SignUp.tsx

```diff
@@ -0,0 +1,128 @@
+
+
+import './styles.css';
+import React, { useState } from 'react';
+import { useSignUp } from '../../hooks/useSignUp';
+import { useNavigate, Link } from 'react-router-dom';
+
+export const SignUp: React.FC = () => {
+  const [name, setName] = useState<string>("");
+  const [middleName, setMiddleName] = useState<string>("");
+  const [surname, setSurname] = useState<string>("");
+  const [secondSurname, setSecondSurname] = useState<string>("");
+  const [birthday, setBirthday] = useState<string>("");
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
```

Comment:
use a reducer here

Reply:
Also, it will help you with validation and showing errors for each field